Add code to generate signature from Method (#1)
invenia/ExprTools.jl: README.md

Alternatively see the [MacroTools](https://github.com/MikeInnes/MacroTools.jl) package for a more powerful set of tools.

Currently, this package provides the `splitdef`, `signature` and `combinedef` functions which are useful for inspecting and manipulating function definition expressions.
 - `splitdef` works on a function definition expression and returns a `Dict` of its parts.
 - `signature` works on a `Method` returning a similar `Dict` that holds the parts of the expressions that would form its signature.

>  - `combinedef` takes such `Dict` and builds it back into an expression

Reviewer: I'd put this right after `splitdef`:
```suggestion
 - `combinedef` takes `Dict` from `splitdef` and builds it into an expression
```

Alternatively see the [MacroTools](https://github.com/MikeInnes/MacroTools.jl) package for a more powerful set of tools.

Currently, this package provides the `splitdef`, `signature` and `combinedef` functions which are useful for inspecting and manipulating function definition expressions.
 - `splitdef` works on a function definition expression and returns a `Dict` of its parts.

>  - `combinedef` takes `Dict` from `splitdef` and builds it into an expression.

Reviewer 2: ```suggestion
 - `combinedef` takes a `Dict` from `splitdef` and builds it into an expression.
```
